password-update-service-samples/README.md

```diff
@@ -0,0 +1,21 @@
+# Password Update Service Samples
```

Comment:
Shall we say Pre Update Password Service Samples ?

Let's term the directory similarly -> pre-update-password-service-samples

Reply:
Fixed with 0a6405027d81ee3b63b9bc16b80ededc1973d230